active_directory_ldap/komand_active_directory_ldap/connection/connection.py

```diff
@@ -20,7 +20,10 @@ def connect(self, params):
         port = params.get(Input.PORT)
         user_name = params.get(Input.USERNAME_PASSWORD).get('username')
         password = params.get(Input.USERNAME_PASSWORD).get('password')
-        self.logger.info(f'Connecting to {host}{port}')
+        if host.find(':') != -1:
+            host = host.split(':')
+            host = host[0]
```

Comment:
Should we wrap in try/except around a possible IndexError?

Reply:
There is no way to get an index error here. To get to line 24 the string must have a `:` in it. If it dose it will get split. Even if the string is something like `:foo` or even `:` Then host[0] will be a blank string.